### PR TITLE
Adds New FirebaseClientWrapper

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: coda_v2_python_client/firebase_client_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import firebase_admin
+from core_data_modules.logging import Logger
+from firebase_admin import credentials
+from firebase_admin import firestore
+
+
+class FirebaseClientWrapper:
```
Comment (as2388) [inline coda_v2_python_client/firebase_client_wrapper.py:7]: I think this should be called CodaV2Client, assuming you expect this to provide the api for other users.

File: coda_v2_python_client/firebase_client_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        """Inits FirebaseClientWrapper
+
+        Args:
+            Firestore client (google.cloud.firestore.Firestore): Client for interacting with Google Cloud Firestore.
+        """
+        self._client = client
+
+    @classmethod
+    def init_client(cls, crypto_token_path, app_name):
+        """Inits Firestore client
```
Comment (as2388) [inline coda_v2_python_client/firebase_client_wrapper.py:18]: This is "Inits Firestore client" but __init__ says "Inits FirebaseClientWrapper", which suggests these are returning different types when they don't. Please use the same description in both places

File: coda_v2_python_client/firebase_client_wrapper.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+from core_data_modules.logging import Logger
+from firebase_admin import credentials
+from firebase_admin import firestore
+
+
+class FirebaseClientWrapper:
+    def __init__(self, client):
+        """Inits FirebaseClientWrapper
+
+        Args:
```
Comment (as2388) [inline coda_v2_python_client/firebase_client_wrapper.py:11]: Python has multiple valid schemes for filling out docstrings, but I'd suggest using the same format as we use everywhere else at AVF for consistency

File: coda_v2_python_client/firebase_client_wrapper.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            Firestore client (google.cloud.firestore.Firestore): Client for interacting with Google Cloud Firestore.
+        """
+        self._client = client
+
+    @classmethod
+    def init_client(cls, crypto_token_path, app_name):
```
Comment (as2388) [inline coda_v2_python_client/firebase_client_wrapper.py:17]: Please provide a default app name e.g.
```suggestion
    def init_client(cls, crypto_token_path, app_name="CodaV2Client"):
```
This is so initialisation is easy for the common case of only needing to connect to one Coda firestore

**Review by as2388 — APPROVED** [review]

File: coda_v2_python_client/firebase_client_wrapper.py (hunk 3 above)
Comment (DanielMwendwa) [reply coda_v2_python_client/firebase_client_wrapper.py]: thanks, noted.

**Pull request merged**